content/webapp/components/Body/Body.tsx

```diff
@@ -398,124 +412,152 @@ const Body: FunctionComponent<Props> = ({
+              {slice.type === 'gifVideo' && (
+                <SpacingComponent>
                   <Layout10>
                     <GifVideo {...slice.value} />
                   </Layout10>
-                )}
-                {slice.type === 'iframe' && (
+                </SpacingComponent>
+              )}
+              {slice.type === 'iframe' && (
+                <SpacingComponent>
                   <Layout10>
                     <Iframe {...slice.value} />
                   </Layout10>
-                )}
-                {slice.type === 'contact' && (
+                </SpacingComponent>
+              )}
+              {slice.type === 'contact' && (
+                <SpacingComponent>
                   <LayoutWidth width={minWidth}>
                     <Contact {...slice.value} />
                   </LayoutWidth>
-                )}
-                {slice.type === 'collectionVenue' && (
-                  <>
-                    {slice.value.showClosingTimes && (
-                      <LayoutWidth width={minWidth}>
-                        <VenueClosedPeriods venue={slice.value.content} />
-                      </LayoutWidth>
-                    )}
-                    {!slice.value.showClosingTimes && (
-                      <>
-                        <Layout
-                          gridSizes={
-                            slice.weight === 'featured'
-                              ? {
-                                  s: 12,
-                                  m: 12,
-                                  l: 11,
-                                  shiftL: 1,
-                                  xl: 10,
-                                  shiftXL: 2,
-                                }
-                              : {
-                                  s: 12,
-                                  m: 10,
-                                  shiftM: 1,
-                                  l: 8,
-                                  shiftL: 2,
-                                  xl: 8,
-                                  shiftXL: 2,
-                                }
-                          }
-                        >
-                          <VenueHours
-                            venue={slice.value.content}
-                            weight={slice.weight || 'default'}
-                          />
-                        </Layout>
-                      </>
-                    )}
-                  </>
-                )}
-                {slice.type === 'table' && (
+                </SpacingComponent>
+              )}
+              {slice.type === 'collectionVenue' && (
+                <SpacingComponent>
+                  {slice.value.showClosingTimes && (
+                    <LayoutWidth width={minWidth}>
+                      <VenueClosedPeriods venue={slice.value.content} />
+                    </LayoutWidth>
+                  )}
+                  {!slice.value.showClosingTimes && (
+                    <>
```

Comment:
Do we need these, if the Layout is the only element inside? Or can we discard these too?

I wouldn't normally ask, but it looks like you've already binned one pair of `<> … </>` in this branch.

Reply:
Good point – we don't need 'em. I'll get rid.